guides/release/components/helper-functions.md

```diff
@@ -208,9 +208,21 @@ For the full list of available helpers, you can check the [template helpers API
 ### The `get` helper
 
 The [`{{get}}`](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/get?anchor=get)
-helper makes it easy to dynamically send the value of a variable to another
-helper or component. This can be useful if you want to output one of several
-values based on the result of a getter.
+helper makes it easy to dynamically look up a property on an object. The second argument to `{{get}}` can be a string or an integer depending on the object being accessed.
+
+
+accessing properties on an object with string keys:
+```handlebars
+{{get this.someObject "object_key"}}
+```
+accessing via indices:
+
+```handlebars
+{{get this.someObjectWithNumericKeys 0}}
+
+{{get this.someArray 0}}
+```
```

Comment:
```suggestion
accessing properties on an object with numeric keys:

```handlebars
{{get this.someObjectWithNumericKeys 0}}
` ` `

accessing elements in an array:

```handlebars
{{get this.someArray 0}}
` ` `
```
(Note: I had to put spaces between the closing  ```s to keep it from closing the suggestion block)